src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBuilder.java

```diff
@@ -198,7 +195,7 @@ public void serializeToView(@NotNull ByteBuf buf)
         {
             final BuildingBuilderResource resource = neededResources.get(entry.getKey());
             //ByteBufUtils.writeItemStack() is Buggy, serialize itemId and damage separately;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)